game/server/mapzones/models.py

```diff
@@ -28,6 +28,16 @@ class UnculledZoneJs(BaseModel):
     class Config:
         title = "UnculledZone"
 
+    @staticmethod
+    def from_game(game: Game) -> list[UnculledZoneJs]:
+        return [
+            UnculledZoneJs(
+                position=zone.latlng(),
+                radius=game.settings.perf_culling_distance * 1000,
+            )
+            for zone in game.get_culling_zones()
+        ]
+
```

Comment:
Refactored this from `game/server/mapzones/routes.py` since we need this in on line 48 `game/server/game/models.py`